types/react-native/v0.68/index.d.ts

```diff
@@ -39,6 +39,7 @@
 //                 Pedro Hernández <https://github.com/phvillegas>
 //                 Sebastian Silbermann <https://github.com/eps1lon>
 //                 Zihan Chen <https://github.com/ZihanChen-MSFT>
+//                 Saad Najmi <https://github.com/saadnajmi>
```

Comment:
Did you mean to add this to the latest releases as well?

Reply:
Good catch, thanks!